Resources/Prototypes/SimpleStation14/Entites/Weapons/rifle.yml

```diff
@@ -0,0 +1,50 @@
+- type: entity
+  name: G11
+  parent: BaseWeaponRifle
+  id: WeaponRifleG11
+  description: A unique rifle with an clock-like internals and a top fed magazine. Uses .20 rifle ammo. #switch to caseless when you have the time
```

Comment:
```suggestion
  description: A unique rifle with a clock-like internals and a top fed magazine. Uses .20 rifle ammo. # TODO Use caseless ammunition
```

Reply:
this needs to be removed entirely since it is already caseless